pandas/tests/io/test_encoding_aliases.py

```diff
@@ -0,0 +1,32 @@
+import pandas, os, nose
+
```

Comment:
Ok, I added utf-8.  How exactly do I run all the tests once I have it in pandas/io/tests/parser/common.py?  I tried `nosetests pandas/io/tests/parser/common.py` from root and it ran 0 tests.  When I run it with `nosetests pandas/tests/io/test_encoding_aliases.py` it works.